includes/helper/class-wp-job-manager-helper.php

```diff
@@ -545,39 +545,41 @@ function ( $plugin ) use ( $search_term ) {
 	/**
 	 * Return if we should show or not the bulk activation form.
 	 *
-	 * @param array $licenced_plugins The list of licensed plugins to handle.
+	 * @param array $licenced_plugins The list of licenced plugins to handle.
 	 *
 	 * @return bool If we should show the bulk activation form or not.
 	 */
 	private function show_bulk_activation_form( $licenced_plugins ) {
 		foreach ( array_keys( $licenced_plugins ) as $product_slug ) {
 			$licence = self::get_plugin_licence( $product_slug );
-			if ( empty( $licence['licence_key'] ) && apply_filters( 'wpjm_display_license_form_for_addon', true, $product_slug ) ) {
+			if ( empty( $licence['licence_key'] ) && apply_filters( 'wpjm_display_licence_form_for_addon', true, $product_slug ) ) {
 				return true;
 			}
 		}
 		return false;
 	}
 
 	/**
-	 * Outputs unset license key notices.
+	 * Outputs unset licence key notices.
 	 */
-	public function licence_error_notices() {
-		$screen = get_current_screen();
-		if (
-			null === $screen ||
-			in_array( $screen->id, [ 'job_listing_page_job-manager-addons' ], true ) ||
-			! current_user_can( 'update_plugins' )
-		) {
-			return;
-		}
```

Comment:
After speaking with @aaronfc we realised that `null` was being encountered when it ought not have been. This was removed as we decided that (1) error notices and (2) missing license notices should be displayed on the add-ons page.